### ISSUE TITLE
Fire elemental from shamans

### ISSUE THREAD

**Issue by powme56** [SunstriderEmu/BugTracker#3888]
Hello, the fire elemental shamans can summon through their totems has one if spell bugged, the damages from his fire nova are two low, indeed as stated in wow gamepedia : 

"They have two spells, a fire nova, that burns enemies in its radius for approximately 700 damage at 70, and a single-target Fire Blast. Because of their limited mana pool and poor regeneration, it should not be expected that the elemental cast either spell more than 4 times per summon. For the rest of its life, the elemental follows its target and punches them, dealing a varying amount of fire damage. The elemental also has a fiery aura, which burns nearby enemies for small amounts of damage every 2 seconds."

Or on endless the damages from his fire nova is more like 100-200 at max as this screenshot of me testing it can show you : 
![elem](https://user-images.githubusercontent.com/7816439/86660715-44c1e900-bfeb-11ea-952f-7848e22d8845.jpg) where it hit for 183 in this case and still as stated by wow gamepedia : "The Fire Elemental benefits from 55% of its master’s spell power."

I know it is not a huge bug but it stills come very annoying during pvp or big aoe pack in raid where it can be clutch sometimes as morogrim.

Anyway thanks for your time reading that and for your response, keep rocking guys !

**Comment by TheMoneyHater** [comment]
Yeah this needs to be fixed since fire ele is suppose to be insane in openworld pvp with that totem and even in pve.
